hyrax.gemspec

```diff
@@ -49,7 +49,7 @@ SUMMARY
   spec.add_dependency 'flot-rails', '~> 0.0.6'
   spec.add_dependency 'font-awesome-rails', '~> 4.2'
   spec.add_dependency 'hydra-derivatives', '~> 3.3'
-  spec.add_dependency 'hydra-editor', '>= 3.3', '< 5.0'
+  spec.add_dependency 'hydra-editor', '~> 5.0'
```

Comment:
is it possible to do this without dropping `hydra-editor` 3.3 and 4.x support?

Reply:
I've pushed an update ... would the removal of the to_model method in `app/forms/hyrax/forms/workflow_responsibility_form.rb` be a problem?

Reply:
well the tests still pass